src/objects/Test_Score__c.object

```diff
@@ -0,0 +1,168 @@
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>SaveEdit</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Tab</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>View</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <allowInChatterGroups>false</allowInChatterGroups>
+    <compactLayoutAssignment>SYSTEM</compactLayoutAssignment>
+    <deploymentStatus>Deployed</deploymentStatus>
+    <description>The score on a section of a specific test or a composite score.</description>
+    <enableActivities>true</enableActivities>
+    <enableBulkApi>true</enableBulkApi>
+    <enableFeeds>false</enableFeeds>
+    <enableHistory>false</enableHistory>
+    <enableLicensing>false</enableLicensing>
+    <enableReports>true</enableReports>
+    <enableSearch>true</enableSearch>
+    <enableSharing>true</enableSharing>
+    <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Percentile__c</fullName>
+        <description>The percentile ranking for this score.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The percentile ranking for this score.</inlineHelpText>
+        <label>Percentile</label>
+        <precision>6</precision>
+        <required>false</required>
+        <scale>3</scale>
+        <trackTrending>false</trackTrending>
+        <type>Percent</type>
+    </fields>
+    <fields>
+        <fullName>Score__c</fullName>
+        <description>The numeric value of the score received.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The numeric value of the score received.</inlineHelpText>
+        <label>Score</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>5</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Subject_Area__c</fullName>
+        <description>The subject matter or category for this test score.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The subject matter or category for this test score.</inlineHelpText>
+        <label>Subject Area</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <valueSetDefinition>
+                <sorted>true</sorted>
+                <value>
+                    <fullName>Language</fullName>
+                    <default>false</default>
+                    <label>Language</label>
+                </value>
+                <value>
+                    <fullName>Mathematics</fullName>
+                    <default>false</default>
+                    <label>Mathematics</label>
+                </value>
+                <value>
```

Comment:
I thought we were going with Overall at the end? (I honestly don't have a dog in that fight, but I know it was brought up)

Reply:
Yep -- we are. It showed up at the end when I looked at it in Salesforce but for some reason is still alphabetizied here. I'll fix it.